### ISSUE TITLE
Migrate Content Away From Google Code

### ISSUE THREAD

**Issue by ChrisMissal** [chucknorris/roundhouse#177]

**Comment by ChrisMissal** [comment]
@ferventcoder Is this something we still want to do, or should we just leverage the GitHub wiki for all RH content?

**Comment by ferventcoder** [comment]
Yeah, def need to migrate the old downloads (some of them at the very least over to gcode).

**Comment by ChrisMissal** [comment]
I was hoping we could host [the downloads](https://code.google.com/p/roundhouse/downloads/list) as binaries here on GitHub under the Releases section. When a release is created though, it matches the downloadable artifacts to the current commit. That would likely be very confusing for older downloads. Anybody have thoughts on where we could host the older zip files? @ferventcoder @BiggerNoise

**Comment by ferventcoder** [comment]
I don't. I know if you perform the full migration, it will work. Lightbulb! Maybe we perform the full migration to a name like chucknorris/roundhouse-old

**Comment by ChrisMissal** [comment]
The GoogleCode migrator doesn't let you select the new repo name, it is forced to use the same one: "roundhouse". So I created a new account, and migrated it, but it failed with little to no feedback.

I'm going to attempt it again though and see if I can actually get some meaningful results.

**Comment by ChrisMissal** [comment]
Export started: https://github.com/roundhouse-archive/roundhouse

**Comment by ChrisMissal** [comment]
![image](https://cloud.githubusercontent.com/assets/67798/8993577/7682fcbc-36cb-11e5-9bcf-b6ad2c1ec878.png)

I'll contact support to see if I can get something.

**Comment by ChrisMissal** [comment]
Ok, the export actually finished, I exported to [my fork](/chrismissal/roundhouse), but the export doesn't support downloads like Google Code did.

**Me:**
> It appears that nothing happens with the downloads page (https://code.google.com/p/roundhouse/downloads/list), is that correct?

**Google Support**
> Correct. GitHub does not support arbitrary downloads in the way that Google Code did. However, Github does have "releases".  Releases must be associated with a tag, and will automatically include a download for the source at that tag. You can also add arbitrary other files associated with each release. e.g. Guice's release page which includes the jars for download, for example: https://github.com/google/guice/releases. 

> If you need a standalone download not associated with a release, you can include the file in your gh-pages branch (e.g. https://github.com/google/guice/tree/gh-pages/user-docs) and reference it from a wiki page (e.g. http://google.github.io/guice/user-docs/Guice-1.0-Users-Guide.pdf.)

So we need to determine a new plan.

**Comment by ferventcoder** [comment]
Well that's kind of yuck.

However, is there some way to set up releases for the older versions?

**Comment by ChrisMissal** [comment]
Ok, we can do this.

https://github.com/ChrisMissal/roundhouse/releases/tag/v0.0.0.54

We just need to identify the commit we want to associate with that release. There are 25 of them in Google Code.

**Comment by ferventcoder** [comment]
@ChrisMissal you sir are awesome! :)

**Comment by ChrisMissal** [comment]
I'm going to open a new ticket to track getting this work done.
